libvips/foreign/dzsave.c

```diff
@@ -326,6 +326,14 @@ typedef struct _VipsGsfDirectory {
 	 */
         GsfOutput *container;
 
+	/* Track number of files for ZIP64 limit check.
+	 */
+	unsigned int file_count;
```

Comment:
Actually the ZIP64 format uses uint64 for the number of entries in thr ZIP64 EOCD, so this should probably be size_t as well.